src/invidious/views/add_playlist_items.ecr

```diff
@@ -29,6 +29,7 @@
 }.to_pretty_json
 %>
 </script>
+<script src="/js/_helpers.js?v=<%= ASSET_COMMIT %>"></script>
```

Comment:
Note that if you add this to the `<head>` element in `template.ecr`, you don't need to add it everywhere else ^^

Reply:
Oh, includes are slightly complicated. I've hoped it will be more simple 😸

The most offensive is `_helpers.js` doesn't import anyway (https://test.invidious.io/watch?v=cwyTleTL06Y)
![image](https://user-images.githubusercontent.com/24810600/168477616-442c166f-806d-4724-b7f2-6ee4cd2f6b61.png)